### ISSUE TITLE
Re-enable public wiki editing

### ISSUE THREAD

**Issue by peterflynn** [adobe/brackets#10511]
See https://groups.google.com/forum/#!topic/brackets-dev/zz239wrH-rI -- due to a string of spam attacks, we've made the wiki read-only so only committers can modify it.

This should be _temporary_, so this bug is a reminder to try re-enabling public write access in the near future.  Ideally, we'll just hear from GitHub support that they have better spam protection in place.  If we don't hear anything, in a few weeks or a month we should try re-enabling access anyway to test the waters and see if the situation has improved.

**Comment by busykai** [comment]
I've reported abuse, by the way, so hopefully others did as well and it would call for github team attention.

**Comment by peterflynn** [comment]
@busykai Yep, me too. What I heard back was basically 'we're working on a way to block spammers like this but we don't have a solution yet' -- so it's definitely on their radar, just no clear ETA yet for a fix or mitigation.

**Comment by MarcelGerber** [comment]
I contacted GitHub as well and what I've got back is basically the same you got, but they also deleted the latest spam commits (see https://github.com/adobe/brackets/wiki/_history).
